gcb/stage/cloudbuild.yaml

```diff
@@ -57,7 +57,7 @@ steps:
   - "--basedir=/workspace"
   - "--tmpdir=/workspace/tmp"
 
-- name: k8s.gcr.io/kube-cross:local
+- name: gcr.io/$PROJECT_ID/kube-cross:local
```

Comment:
See [the comment on 626][626]: I think we want/need to keep the prod image here. I guess we can change the local name in [626] to any random name and we just need to use the same name here, however we need to make sure we use the correct == prod image for this step.


[626]: https://github.com/kubernetes/release/pull/1077/files#diff-5cf63765a3143c845e1695aeb8d41e2eR626